Dockerfile

```diff
@@ -1,17 +1,17 @@
 # Copyright 2019 Adevinta
 
 ARG rails_env=production
-FROM ruby:2.5.9-alpine AS pre-builder
+FROM ruby:3.1-alpine3.15 AS pre-builder
```

Comment:
I would suggest to fix patch version as well:
```
3.1.2-alpine3.15
```
So local and CI matches the version.

Reply:
yes I agree it's better.